mathics_scanner/data/named-characters.yml

```diff
@@ -2741,6 +2741,14 @@ Factorial:
   unicode-equivalent-name: EXCLAMATION MARK
   wl-unicode: "!"
   wl-unicode-name: EXCLAMATION MARK
+Factorial2:
+  ascii: "!!"
+  has-unicode-inverse: false
+  is-letter-like: false
+  operator-name: Factorial2
+  precedence: 610
+  unicode-equivalent: "!!"
```

Comment:
The comments at the top say unicode-equivalent is optional and there are other entries like `++` that omit the wl-unicode and unicode-equivalent. 

Do you have a particular reason in mind for including this?

Reply:
No, I just saw this in factorial and put it here too. Should I remove it?

Reply:
Thanks - yes. And maybe then we should remove it in Factoral as well (and other places this kind of thing exists.

At some point I may add a check that unicode equivalent should not exist if it is the same thing as ascii.